DSCResources/MSFT_AdcsAiaExtension/en-US/MSFT_AdcsAiaExtension.strings.psd1

```diff
@@ -0,0 +1,11 @@
+ConvertFrom-StringData @'
+    AdditionalAiaPathsFound      = Not in desired state, additional AIA URI paths found, "{0}", running Set.
+    AddAiaUriPaths               = Adding "{0}" to desired AIA URI paths.
+    DesiredAiaPathsMissing       = Not in desired state, missing desired AIA URI paths, "{0}", running Set.
+    EnsureAbsentButUriPathsExist = Not in desired state, AIA URI paths should be Absent, but found "{0}".
+    GetAiaUriPaths               = Getting all AIA URI paths.
+    AiaPathsNull                 = Not in desired state, AIA URI paths empty, missing "{0}", running Set.
+    RemoveAiaUriPaths            = Removing "{0}" AIA URI paths.
+    RestartService               = Preparing to restart the CertSvc for changes to take affect.
+    IncorrectAiaUriFound         = Not in desired state, AIA URI path incorrect, found "{0}", running Set.
```

Comment:
Might want to reword this. If the DSC configuration is set to Apply and Monitor it would not run Set if the setting is change to an incorrect setting.